examples/kubernetes/tls-passthrough.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: passthrough-echoserver
+  labels:
+    run: passthrough-echoserver
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+      protocol: TCP
+  selector:
+    run: passthrough-echoserver
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: passthrough-echoserver
+spec:
+  selector:
+    matchLabels:
+      run: passthrough-echoserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: passthrough-echoserver
+    spec:
+      containers:
+        - name: passthrough-echoserver
+          image: gcr.io/k8s-staging-ingressconformance/echoserver:v20220815-e21d1a4
+          ports:
+            - containerPort: 8443
+          env:
```

Comment:
Can you add the pod and namespace env vars too?

xref: https://github.com/envoyproxy/gateway/blob/main/examples/kubernetes/quickstart.yaml#L62-L70